kokkos.presets.json

```diff
@@ -0,0 +1,68 @@
+{
+    "version"   : 3,
+    "cmakeMinimumRequired" : {
+        "major" : 3,
+        "minor" : 22,
+        "patch" : 0
+    },
+    "configurePresets" : [
+        {
+            "name"          : "default",
+            "displayName"   : "default",
+            "binaryDir"     : "${sourceDir}/build-with-${presetName}",
+            "cacheVariables" : {
+                "CMAKE_EXPORT_COMPILE_COMMANDS" : {"type" : "BOOL"  , "value" : "ON"},
+                "CMAKE_BUILD_TYPE"              : {"type" : "STRING", "value" : "RELEASE"},
+                "CMAKE_CXX_STANDARD"            : {"type" : "STRING", "value" : "17"},
+                "CMAKE_CXX_EXTENSIONS"          : {"type" : "BOOL"  , "value" : "OFF"},
+                "BUILD_SHARED_LIBS"             : {"type" : "BOOL"  , "value" : "ON"}
+            }
+        },
+        {
+            "name"        : "GNU-OpenMP",
+            "displayName" : "GNU-OpenMP",
+            "inherits"    : "default",
+            "cacheVariables" : {
+                "Kokkos_ENABLE_OPENMP" : {"type" : "BOOL"  , "value" : "ON"}
```

Comment:
I am not a big fan of the verbose syntax when we can get away with `"Kokkos_ENABLE_OPENMP": "ON"` but not blocking if others feel strongly about it.

Reply:
Ok, I removed these `"type"` and `"value"` ;)